pkg/operator/clustermembercontroller/clustermembercontroller.go

```diff
@@ -61,10 +71,12 @@ func NewClusterMemberController(
 		machineAPIChecker:     machineAPIChecker,
 		etcdClient:            etcdClient,
 		podLister:             kubeInformers.InformersFor(operatorclient.TargetNamespace).Core().V1().Pods().Lister(),
-		nodeLister:            kubeInformers.InformersFor("").Core().V1().Nodes().Lister(),
 		networkLister:         networkInformer.Lister(),
 		masterMachineLister:   machinelistersv1beta1.NewMachineLister(masterMachineInformer.GetIndexer()),
 		masterMachineSelector: masterMachineSelector,
+		masterNodeLister:      corev1listers.NewNodeLister(masterNodeInformer.GetIndexer()),
+		masterNodeSelector:    masterNodeSelector,
+		configMapLister:       kubeInformers.InformersFor(operatorclient.TargetNamespace).Core().V1().ConfigMaps().Lister().ConfigMaps(operatorclient.TargetNamespace),
 	}
 	return factory.New().ResyncEvery(time.Minute).
```

Comment:
shouldn't we also add `kubeInformers.InformersFor(operatorclient.TargetNamespace).Core().V1().ConfigMaps()` to the `WithInformer` method?